### ISSUE TITLE
Small wishlist of (10) mods for compatibility

### ISSUE THREAD

**Issue by SpiderKolo** [Corail31/woodcutter#18]
Let me tell you first that I understand if making any incompatibilities might be too hard to spend time on as mods tend to update often. It's also hard to follow all of those mods with the updates.
That being said: there are number of mods which I feel would greatly benefit from having Sawmill made of the wood from that mod.
Not just mods which adds new trees or new "trees" - also ones which add new planks crafted in specific way.
I will be listing all mods and list their trees and planks they introduce.

Atmospheric: https://www.curseforge.com/minecraft/mc-mods/atmospheric
This mod introduces 4 new types of Trees: **Aspen, Kousa, Yucca** and **Rosewood**.
Mod itself introduces (or is suppose to introduce) many tropical biomes ot the game.

**Autumnity:** https://www.curseforge.com/minecraft/mc-mods/autumnity
This mod introduces just 1 new type of tree - **Maple tree.**
This tree have multiple leaf colors and corresponding to them saplings, but that doesn't matter for the contest of this mod.

Bamboo Blocks: https://www.curseforge.com/minecraft/mc-mods/bamboo-blocks
No new trees, but it adds new type of **planks made of Bamboo**, as well as **Thatch**...
While I'm not sure if Thatch should be something you make base of for Sawmill, it could be at least cut into stairs and slabs (Thatch is made of Bamboo and Sugar Cane - something you could easily cut using iron Saw).

**Bloomful:** https://www.curseforge.com/minecraft/mc-mods/bloomful
Similarly to Autumnity - one new type of tree - **Wisteria** with many leaf coloration and fitting saplings.
Let me tell you tho - this tree and blocks made of them looks absolutely gorgeous.

**Buzzier Bees:** https://www.curseforge.com/minecraft/mc-mods/buzzier-bees
No new types of tree, but one new type of planks - **Hive Planks.**
Since your Sawmills are made of Planks, I decided to include this mod into list as well.

Swamp Expansion: https://www.curseforge.com/minecraft/mc-mods/swamp-expansion
One, kinda familiar new type of tree - **Willow.**
You already added Willow Sawmill from Biomes O'Plenty, yes, however this mod introduces also many other cool features slightly better than Biomes for the swamps. Well... this mod is focused on them after all...

The Endergetic Expansion: https://www.curseforge.com/minecraft/mc-mods/endergetic
Well... new type of tr- oh wait... not a tree! a mushroom - **Poise**.
It's an End-game block you have access after killing Ender Dragon and reaching outer lands of the End.

**Upgrade Aquatic:** https://www.curseforge.com/minecraft/mc-mods/upgrade-aquatic
This mod does introduce new type of tree... and actually two new types of wood:
**River** (with sapling and leafes) and **Driftwood** (no saplings or leafes - it's drifting on open oceans.

**Quark:** https://www.curseforge.com/minecraft/mc-mods/quark
This mod have the most options for compatibility from the bunch I mentioned so far.
No new trees, but a lot of new Plank types... **16** to be exact.
This mod adds **Dyeable Planks,** Vertical Planks, Vertical Plank Slabs...
With 16 new colors of Planks that could mean a lot of new cool colors of Sawmills. And also plenty of recipes.
I haven't seen specific types of Doors or Fences for those planks yet tho... we have just Slabs and Stairs (well... and Vertical Slabs...).
There are also stacked, "compressed" blocks of items which are often taking a lot of space and are easily farmable: Carrots, Potatoes, Sugar Cane... and also - **Bamboo.**
While Bamboo in this mod alone on it's own doesn't produce anything but Sticks, it could include few recipes for blocks from **Bamboo Blocks mod** I mentioned few mods ago.
There is also new type of blocks which could maybe be included as a block you could cut on sawmills - **Midori.** Midori is Block made out of Cactus Paste (You cook Cactus in order to get the Paste, and then use Paste to get the Dye).
I don't think there should be Sawmill made of Midori as it's base, but I think it could make sense if you could shape Midori on Sawmill.
There is also another type of **Thatch** introduced there - this block is made out of Wheat instead and can be used to reduce fall damage. It also does have few standard shapes for building (Well... three of - Slab, Vertical Slab and Stairs).
As a honorable mention - there is cool, entirely made out of wood block called **Feeding Trough.**
It's in honorable mentions because there might be too much wood needed in order to create this block on Sawmill.

**Extended Mushrooms:** https://www.curseforge.com/minecraft/mc-mods/extended-mushrooms
Again no new types of tree, but instead we have two new types of Mushroom with recipies similar to Trees and tweaks for our Vanilla Mushrooms to make them also have those recipes.
New types include: **Poisonous Mushroom and Glowshrooms**
New types of Planks: **Poisonous Mushroom, Glowshrooms and "Normal" Brown and Red Mushrooms**
(Brown and Red Mushrooms share their Stem block, which is used for new planks).
Interestingly enough, creator of this mod already included recipes for your Sawmill, so all it's probably left are Sawmills themself.

After adding all of the mentioned Sawmills from all of those mods, there would be total of 28 new Sawmills, 16 of which are just form Quark planks and other 12 are form the rest of the mods I mentioned.

Honorable Mentions:
**Extra Boats:** https://www.curseforge.com/minecraft/mc-mods/extra-boats
This mods add three new types of Boats for every type of wood in base game and few mods.
One of which is fully made out of planks (it's just bigger so it can fit up to 4 entities).
It's Honorable Mention because... well, that one boat might be too big for recipe in your sawmill really...
We already have small boat from one log from your sawmill which is a stretch, there is no way we could include recipe for second, twice as big boat (since every recipe includes one Plank or one Log).

**Comment by SpiderKolo** [comment]
Small Recipe Suggestion: I was thinking if maybe ability to turn 1 Bamboo into 1 Scaffolding wouldn't be nice...
Would it be overpowered? I don't know... recipe does require String as well... It's up to you to decide.
I though I would throw in this one as well since I mentioned Bamboo Blocks support.

**Comment by Corail31** [comment]
Hi SpiderKolo,
The woodcutter provides a type of recipes that can be used by any mod.
As you imagine i prefer to create code than following the changes of block in multiple mods, not to mention the time it would take.
It could be added directly in theses mods, or with third party mod doing the job like it's with pjlasl on fabric.
